src/main/java/fi/metatavu/metamind/bot/functions/KuntaApiGetEventsMetaBotFunction.java

```diff
@@ -0,0 +1,190 @@
+package fi.metatavu.metamind.bot.functions;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.ProtocolException;
+import java.net.URL;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import com.bladecoder.ink.runtime.Story;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.rabidgremlin.mutters.bot.ink.CurrentResponse;
+import com.rabidgremlin.mutters.bot.ink.functions.FunctionDetails;
+import com.rabidgremlin.mutters.bot.ink.functions.FunctionHelper;
+import com.rabidgremlin.mutters.core.IntentMatch;
+import com.rabidgremlin.mutters.core.session.Session;
+
+import fi.metatavu.metamind.kuntaapi.Events;
+
+/**
+ * Metabot function to get events from KuntaAPI
+ * 
+ * @author Ville Koivukangas
+ */
+@ApplicationScoped
+public class KuntaApiGetEventsMetaBotFunction extends AbstractMetaBotFunction {
+  @Inject
+  private Logger logger;
+
+  @Override
+  public String getFunctionName() {
+    return "KUNTA_API_GET_EVENTS";
+  }
+
+  @Override
+  public void execute(CurrentResponse currentResponse, Session session, IntentMatch intentMatch, Story story,
+      String param) {
+    FunctionDetails details = FunctionHelper.parseFunctionString(param);
+
+    String authorizationToken = (String) session.getLongTermAttribute("authorizationToken");
+    String organizationId = details.getFunctionParams().get("organizationId");
+    String route = "events";
+    String routeId = "";
+
+    ArrayList<Events> events = new ArrayList<Events>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)